### PR TITLE
chore: bump ory sdk v1.1.0

### PR DESCRIPTION
**Pull request by Benehiko**
This PR is a cherry pick with some cleanup from https://github.com/ory/kratos-selfservice-ui-node/pull/228/commits/473d7feed3862407f2c8c616205f4e1f2b6cd6ad

## Checklist

- [ ] I have read the [contributing guidelines](../blob/master/CONTRIBUTING.md) and signed the CLA.
- [ ] I have referenced an issue containing the design document if my change introduces a new feature.
- [ ] I have read the [security policy](../security/policy).
- [ ] I confirm that this pull request does not address a security vulnerability. 
      If this pull request addresses a security vulnerability, 
      I confirm that I got approval (please contact [security@ory.sh](mailto:security@ory.sh)) from the maintainers to push the changes.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [ ] I have added the necessary documentation within the code base (if appropriate).

### CODE REVIEW

**Comment by DrBu7cher** [comment]
Also, please make sure, that "we" really want the change I made to the Makefile. I did this for easier testing and replacing the upstream for the docker file, but since I included the required file in the `.gitignore`, simply running the `make docker` command would not work without this `.docker` file.
I hope this does not break pipelines etc?

**Comment by DrBu7cher** [comment]
We can easily revert this, it doesn't do anything regarding the real "version bumping purpose" of this and my PR. It was just an idea from me, to easily adjust the docker upstream for local testing purposes.
We could remove the gitignore entry and add a .docker or .docker-upstream file? This is up to you to decide or maybe it's a topic of a different PR then.

**Comment by Benehiko** [comment]
> We can easily revert this, it doesn't do anything regarding the real "version bumping purpose" of this and my PR. It was just an idea from me, to easily adjust the docker upstream for local testing purposes. We could remove the gitignore entry and add a .docker or .docker-upstream file? This is up to you to decide or maybe it's a topic of a different PR then.

Hmm good point, yeah let's revert this file then for now

File: Makefile [hunk 1]
```diff
@@ -1,12 +1,15 @@
 rand := $(shell openssl rand -hex 6)
+ROOT_DIR := $(dir $(realpath $(lastword $(MAKEFILE_LIST))))
+DOCKER_UPSTREAM ?= $(shell cat $(ROOT_DIR)/.docker 2> /dev/null)
+DOCKER ?= $(shell which podman || which docker)
```
Comment (alnr) [inline Makefile:4]:
```suggestion
DOCKER_UPSTREAM := oryd/kratos-selfservice-ui-node
DOCKER := docker
```
Delete `.docker` dir and mention in the readme to use
```
make DOCKER=podman DOCKER_UPSTREAM=gcr.io/my/kratos
```

File: .nvmrc [hunk 2]
```diff
@@ -0,0 +1 @@
+v18.12.1
```
Comment (alnr) [inline .nvmrc:1]: please add tailing newline

File: Makefile [hunk 3]
```diff
@@ -1,12 +1,15 @@
 rand := $(shell openssl rand -hex 6)
+ROOT_DIR := $(dir $(realpath $(lastword $(MAKEFILE_LIST))))
```
[the remark this answers is not in the document]
Comment (Benehiko) [reply Makefile]: true, i have reverted this file now

File: src/routes/registration.ts [hunk 4]
```diff
@@ -54,9 +54,9 @@ export const createRegistrationRoute: RouteCreator =
       return
     }
 
-    sdk
-      .getSelfServiceRegistrationFlow(flow, req.header("Cookie"))
-      .then(({ data: flow }: { data: SelfServiceRegistrationFlow & any }) => {
+    frontend
+      .getRegistrationFlow({ id: flow, cookie: req.header("Cookie") })
+      .then(({ data: flow }: { data: RegistrationFlow & any }) => {
```
[the remark this answers is not in the document]
Comment (Benehiko) [reply src/routes/registration.ts:59]: good catch!

**Review by zepatrik — APPROVED** [review]
LGTM :+1:

**Pull request merged**